lib/IRGen/GenBuiltin.cpp

```diff
@@ -354,6 +354,31 @@ void irgen::emitBuiltinCall(IRGenFunction &IGF, const BuiltinInfo &Builtin,
     emitBuildOrdinarySerialExecutorRef(IGF, actor, type, conf, out);
     return;
   }
+  
+  if (Builtin.ID == BuiltinValueKind::InitializeDistributedRemoteActor) {
+    auto fn = IGF.IGM.getDistributedActorInitializeRemoteFn();
+    auto actor = args.claimNext();
+    actor = IGF.Builder.CreateBitCast(actor, IGF.IGM.RefCountedPtrTy);
+    // init(resolve address, using transport)
+    // auto address = args.claimNext(); // FIXME: actually take those parameters
+    // auto transport = args.claimNext();
```

Comment:
This initializer is in a weird state -- the initializer we do want to keep working like this because it works by setting the distributed/remote flag on the actor. we use this for all the "remote call" tests which already work.

This depends on the complete rewrite/implementation of the "resolve initializer with initializing the storage enum" that we'll need help with to be the real deal. 

I cleaned up this a bit but would prefer to keep as is and replace with the real implementation, whatever it might be -- perhaps we don't need this operation at all here, and we will simply implement it on the syntax level by checking the case of the `$storage` enum